File: algo_python.py/Bubble_Sort.py
a = []
n = int(input("Enter range: "))

user_input = input(f"Enter {n} numbers separated by spaces: ")
for num in user_input.split():
    a.append(int(num))

print("Before sorting: ", a)

# Bubble Sort Algoritm
for i in range(0, n - 1):
    for j in range(0, n - i - 1):
        # Swap 
        if a[j] > a[j + 1]:
            temp = a[j]
            a[j] = a[j + 1]
            a[j + 1] = temp

print("Sorted: ", a)

File: algo_c++.cpp/Bubble_Sort.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int n;
    vector<int> a;

    // Input the number of elements
    cout << "Enter range: ";
    cin >> n;

    // Input elements into the vector
    cout << "Enter " << n << " numbers: ";
    for (int j = 0; j < n; ++j) {
        int b;
        cin >> b;
        a.push_back(b);
    }

    // Print elements before sorting
    cout << "Before sorting: ";
    for (int num : a) {
        cout << num << " ";
    }
    cout << endl;

    // Bubble Sort algorithm
    for (int i = 0; i < n - 1; ++i) {
        for (int j = 0; j < n - i - 1; ++j) {
            if (a[j] > a[j + 1]) {
                // Swap elements
                int temp = a[j];
                a[j] = a[j + 1];
                a[j + 1] = temp;
            }
        }
    }

    // Print elements after sorting
    cout << "Sorted: ";
    for (int num : a) {
        cout << num << " ";
    }
    cout << endl;

    return 0;
}
